diff --git a/node_modules/react-native-fast-encoder/.bun-tag-c1e57f2cb10cfd4c b/.bun-tag-c1e57f2cb10cfd4c
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/ios/FastEncoderModule.h b/ios/FastEncoderModule.h
index cd701ca786fa0328d65b42ddac584449172f6eec..a87ef59a878c90da69602fedfb052e3b5a691ece 100644
--- a/ios/FastEncoderModule.h
+++ b/ios/FastEncoderModule.h
@@ -2,13 +2,8 @@
 #import "react-native-fast-encoder.h"
 #endif
 
-#ifdef RCT_NEW_ARCH_ENABLED
-  #import "generated/FastEncoderSpec/FastEncoderSpec.h"
-  @interface FastEncoderModule : NSObject <NativeFastEncoderModuleSpec>
-#else
-  #import <React/RCTBridge.h>
-  @interface FastEncoderModule : NSObject <RCTBridgeModule>
-#endif
+#import "generated/FastEncoderSpec/FastEncoderSpec.h"
+@interface FastEncoderModule : NSObject <NativeFastEncoderModuleSpec>
 @property (nonatomic, assign) BOOL setBridgeOnMainQueue;
 
 @end
diff --git a/ios/FastEncoderModule.mm b/ios/FastEncoderModule.mm
index 281574702e2c80083ea63b839524c099a635f5d4..222e68a211f90c98b6ba5b0089974487dea55947 100644
--- a/ios/FastEncoderModule.mm
+++ b/ios/FastEncoderModule.mm
@@ -5,44 +5,30 @@
 #import <jsi/jsi.h>
 
 @implementation FastEncoderModule
-#import <React/RCTUtils.h>
 
 @synthesize bridge = _bridge;
 @synthesize methodQueue = _methodQueue;
-RCT_EXPORT_MODULE() 
-#ifdef NEW_ARCH_ENABLED
-static facebook::jsi::Value install(facebook::jsi::Runtime& runtime,
-                                  facebook::react::TurboModule& turboModule,
-                                  const facebook::jsi::Value* args,
-                                  size_t count)
-{
-    try {
-        fastEncoder::install(runtime);
-        return jsi::Value(true);
-    } catch (const std::exception& e) {
-        return jsi::Value(false);
-    }
-}
-#else
+RCT_EXPORT_MODULE()
+
 RCT_EXPORT_BLOCKING_SYNCHRONOUS_METHOD(install)
 {
     RCTCxxBridge *cxxBridge = (RCTCxxBridge *)_bridge;
     if (cxxBridge == nil) {
         return @false;
     }
-    
+
     auto jsiRuntime = (facebook::jsi::Runtime *)cxxBridge.runtime;
     if (jsiRuntime == nil) {
         return @false;
     }
-    
+
     auto &runtime = *jsiRuntime;
     auto callInvoker = _bridge.jsCallInvoker;
     fastEncoder::install(runtime, callInvoker);
     NSNumber * val = [NSNumber numberWithBool:TRUE];
     return val;
 }
-#endif
+
 + (BOOL)requiresMainQueueSetup {
     return YES;
 }
@@ -57,7 +43,6 @@ - (void)invalidate {
     fastEncoder::cleanup();
 }
 
-
 - (std::shared_ptr<facebook::react::TurboModule>)getTurboModule:
     (const facebook::react::ObjCTurboModule::InitParams &)params
 {
